### PR TITLE
Add styling and html templating to php, ruby, python, and node

### PR DESCRIPTION
**Pull request by emmac3**
Adding styling to the oauth apps for php, ruby, python and node
before:
localhost_8000_request_token php
now:
localhost_8000_request_token
localhost_8000_callback_code=sandbox-sq0cgb-KPyXXQUsC-E9b09lJTb8vA response_type=code state=cfa33ab2e893a5ebaf75d0ef64106291

I left out java and python chalice because:
1. I still don't quite understand python chalice
2. it's a bit harder to serve static files like css in java

### CODE REVIEW

File: connect-examples/oauth/node/index.js [hunk 1]
```diff
@@ -106,64 +115,82 @@ app.get("/request_token", (req, res) => {
...
+    content = messages.displayStateError();
+    res.render('base', {
+      content: content
+    });
+  }
+
+  else if (req.query['error']) {
+    // Check to see if the seller clicked the Deny button and handle it as a special case.
+    if (("access_denied" === req.query['error']) && ("user_denied" === req.query["error_description"])) {
+      res.render(messages.displayError("Authorization denied", "You chose to deny access to the app."));
```
Comment (barakj) [inline connect-examples/oauth/node/index.js:130]: I'm confused about this path. When does the seller get to the Accept/Deny permissions screen? I wasn't able to reach it when running locally with new accounts.

File: connect-examples/oauth/node/index.js [hunk 2]
```diff
@@ -106,64 +115,82 @@ app.get("/request_token", (req, res) => {
...
+        content = messages.displayError('Exception', JSON.stringify(error.result))
+        res.render('base', {
+          content: content
+        });
+      } else {
+        content = messages.displayError('Exception', JSON.stringify(error))
+        res.render('base', {
+          content: content
+        });
+      }
```
Comment (barakj) [inline connect-examples/oauth/node/index.js:182]: Minor: This section feels a bit duplicated, can probably be simplified to something like this:
```
content = messages.displayError('Exception', JSON.stringify((error instanceof ApiError) ? error.result : error));
res.render('base', {
  content: content
});
```

File: connect-examples/oauth/node/package.json [hunk 3]
```diff
@@ -13,6 +13,7 @@
     "dotenv": "^8.2.0",
     "express": "*",
     "md5": "*",
-    "square": "^8.0.0"
+    "square": "^8.0.0",
```
Comment (barakj) [inline connect-examples/oauth/node/package.json:16]: Why are we not using the newest SDK?

File: connect-examples/oauth/ruby/oauth-flow.rb [hunk 4]
```diff
@@ -63,18 +72,49 @@
     # Extract the returned access token from the ObtainTokenResponse object
     if response.success?
 
-      # Here, instead of printing the access token, your application server should store it securely
+      # In production, instead of printing the access token, your application server should store it securely
```
Comment (barakj) [inline connect-examples/oauth/ruby/oauth-flow.rb:75]: Might want to add a similar comment fix in `oauth-flow.py`

**Comment by barakj** [comment]
Not sure how sensitive the `OAuth` token is considered to be, but you have a screenshot of your `OAuth` token in a public repo PR. If it is considered sensitive, maybe should blur out the actual values in the image.

File: connect-examples/oauth/node/package.json (hunk 3 above)
Comment (emmac3) [reply connect-examples/oauth/node/package.json:16]: that code hasn't been merged yet

**Pull request merged**